CHANGELOG.md

```diff
@@ -8,6 +8,27 @@ Changelog
 ### Changed
 
 ### Removed
+* Deprecated `vlan` private-vlan properties:
```

Comment:
Can you add a note to indicate which `node_utils` objects should be used for `pvlan` capabilities?

Reply:
done